### PR TITLE
Hawkular APM 0.11 with opentracing and alerts integration

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

File: src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+Finally, once the order has been confirmed, we associate some additional information with the top level span, before calling the `finish()` method.
+
+This gives a brief insight into how an application can be instrumented to use the opentracing API. The _OrderManager_ example is slightly more complex, in that it also instruments a subsequent publish of the order confirmation on the event bus, and relates the instrumentation back to the original trace using the _FOLLOWS_FROM_ reference type. For more information please see the https://github.com/hawkular/hawkular-apm/tree/0.11.0.Final/examples/vertx-opentracing[code].
+
+
+=== Analysing the Captured Information
+
+When starting the Hawkular APM console, the first page that is displayed shows details for the spans associated with the tracing information that has been captured from the application. The spans can represent _Consumers_ that receive service requests, _Producers_ that make service requests, or internal _Components_ (e.g. databases).
+
+As you can see, the values in the _Operation_ field have the values that were used when building the spans in the vertx application.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc]: field -> column  ?

File: src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,133 @@
+= Hawkular APM supports Opentracing and Alerts
+Gary Brown
+2016-10-17
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, microservice, vertx
+
+The link:https://github.com/hawkular/hawkular-apm/releases/tag/0.11.0.Final[latest Hawkular APM release] includes two new features, a Java based http://opentracing.io/[Opentracing] provider and integration with http://www.hawkular.org/community/docs/developer-guide/alerts.html[Hawkular Alerts].
+
+A video demonstrating these new features can also be viewed https://youtu.be/HSWSiww07RE[here]. The video starts with an explanation of how a vertx application is instrumented using opentracing. If you want to skip that explanation, then start https://youtu.be/HSWSiww07RE?t=7m47s[here for an explanation of the alert trigger] or start https://youtu.be/HSWSiww07RE?t=9m20s[here for the actual demo].
```
Comment (pavolloffay) [inline src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc]: Shouldn't we use OpenTracing?

File: src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+The additional tags we have specified are for information purposes. The `http.url` is used to identify two things, that the endpoint type is HTTP, and secondly the URL is _/orders_. The `service` tag is used to add information on the dependency graph, and finally the `transaction` enables the trace instance to be classified as belonging to this business transaction.
+
+As this service is then making calls out to other services, we want to create a span to encapsulate the service invocation, and identify it as being a child span of the previously created span (i.e. to make it clear that this service invocation is being performed in the scope of the existing trace instance). The follow span represents the call out to the _AccountManager_ to get account information.
+
+```
+        Span getAccountSpan = tracer.buildSpan("GetAccount")
+                .asChildOf(ordersConsumerSpan)
+                .start();
```
Comment (pavolloffay) [inline src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc:43]: maybe put an empty line here

File: src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        tracer.inject(getAccountSpan.context(), Format.Builtin.TEXT_MAP,
+                new VertxMessageInjectAdapter(order));
+
+        eb.send("AccountManager.getAccount", order, acctresp -> {
+
```
Comment (pavolloffay) [inline src/main/jbake/content/blog/2016/10/17/hawkular-apm-opentracing-and-alerts.adoc]: and remove this line

**Review by pavolloffay — COMMENTED** [review]
LGTM maybe add titles to the images, but it's ok as it is.

**Comment by objectiser** [comment]
@theute @pilhuhn Could you merge this please?

**Pull request merged**